pkg/cluster/ports/ports.go

```diff
@@ -27,9 +27,6 @@ const (
 	KubeletPort = 10250
 	// KubeletReadOnlyPort exposes basic read-only services from the kubelet.
 	// May be overridden by a flag at startup.
-	// This is necessary for heapster to collect monitoring stats from the kubelet
-	// until heapster can transition to using the SSL endpoint.
-	// TODO(roberthbailey): Remove this once we have a better solution for heapster.
 	KubeletReadOnlyPort = 10255
```

Comment:
@solodov ideally we need to add clarification where it is used if at all. If it does, this PR https://github.com/kubernetes/kubernetes/pull/100335/ may break things